.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: check-merge-conflict
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+
+-   repo: https://github.com/pre-commit/mirrors-yapf
+    rev: v0.31.0
```

Comment:
DeepChem uses yapf 0.22 - refer [this](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html) of docs.

Reply:
Addressed in 5ea8484